docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -7,6 +7,12 @@ type: guides
 category: configuration
 language: en
 abstract: Learn how to use Meshery Designs effectively
+redirect_from:
+  - /tasks/patterns.md
+  - /reference/mesheryctl
+  - /guides/deploying-sample-apps
+  - /deploying-sample-apps.md
+  - /performance-management.md
```

Comment:
```suggestion
```

We do have dedicated pages for each of these redirects, so ideally we wouldn't want a redirect from these links to the current page that is `working with designs`

Reply:
Okay..Makes sense!
Thanks!